back/engines/commercial/report_builder/app/services/report_builder/queries/single_idea.rb

```diff
@@ -0,0 +1,20 @@
+module ReportBuilder
+  class Queries::SingleIdea < ReportBuilder::Queries::Base
```

Comment:
I've left the name "Single idea" (not "Idea") to make the difference with Most reacted ideas clearer. And I've made the default title text blank because neither "Single idea" nor "Idea" makes a lot of sense.